README.md

```diff
@@ -2,12 +2,37 @@
 
 [![build status](https://img.shields.io/travis/enkidevs/approve-ci/master.svg?style=flat-square)](https://travis-ci.org/enkidevs/approve-ci)
 
-https://developer.github.com/guides/building-a-ci-server/
+Approve-ci monitors pull requests and checks for approval. Once the number of approvals exceeds the number of disapprovals by a set amount the pull request is marked as 'approved' and is ready to merge.
 
-https://developer.github.com/v3/repos/hooks/
+![approve-ci in use](http://i.imgur.com/2aMhuzk.png)
 
-https://developer.github.com/v3/activity/events/types/#pullrequestreviewcommentevent
+## Deployment
 
-https://github.com/mikedeboer/node-github
+To run an approve-ci bot clone this repository, modify `.approve-ci` and deploy your application. You must set a GitHub organisation, repository and token as environment variables, along with the URL of your deployment.
```

Comment:
Also, it would be nice to explain how to protect the branches from being merge before approval: https://github.com/enkidevs/approve-ci/settings/branches